data/json/mapgen/abandoned_warehouse.json

```diff
@@ -194,7 +193,6 @@
     },
     "om_terrain": "abandonedwarehouse",
     "type": "mapgen",
-    "flags": [ "SIDEWALK" ],
```

Comment:
Sidewalk flag was redundant here as it is already defined for abandonedwarehouse in overmap_terrain.

Reply:
Ok, apparently I didn't update the overmap_terrain with the sidewalk flag before, but I just did in commit [8c573f6](https://github.com/CleverRaven/Cataclysm-DDA/pull/25070/commits/8c573f6bfdacf9c005e1e4a9863da21697ac6621). I also moved sidewalk flags to overmap_terrain for a few other locations (of mine) that use it on ALL variations of said locations.